File: docs/examples/special-exports-example-3.zig
pub const Error = error{fell_victim_to_micro_aggression};

pub const Struct = struct {
    number1: i32,
    number2: i32,
};
pub const StructEU = Error!Struct;
pub const StructO = ?Struct;

pub const Union = union(enum) {
    number1: i32,
    number2: i64,
};

File: docs/examples/special-exports-example-3.js
import {
    __zigar,
    Error,
    Struct,
    StructEU,
    StructO,
    Union
} from './special-exports-example-3.zig';
const { typeOf } = __zigar;

console.log(typeOf(Error));
console.log(typeOf(Struct));
console.log(typeOf(StructEU));
console.log(typeOf(StructO));
console.log(typeOf(Union));
